deploy/salt/roots/salt/memex-explorer.sls

```diff
@@ -14,12 +14,9 @@ local_settings_path:
      - name: LOCAL_SETTINGS_PATH
      - value: /vagrant/source/memex/local_settings.py
 
-
-# TODO: The create_apps_Tika_ES_Kibana is not idempotent.  This needs
-# to be fixed or the command needs to be protected so that it only
-# executes when the applications need to be created.
-# As a workaround, the SQL database is currently reset on each provision.
-
+# We originally reset the database on each reprovision because of a database-modifying function that was not idempotent.
+# This may not be necessary.
+# @ahmadia, is there any other reason for it?
```

Comment:
I agree that I ought to have flagged it outside of the code, but I think also having a comment here to mark something for future removal is a good idea. See also https://github.com/memex-explorer/memex-explorer/blob/amfarrell/cleanup-files/source/base/models.py#L135, which should be more specific.

I've been thinking about this in terms of Chesterton's Fence (http://www.chesterton.org/taking-a-fence-down/)